tools/Azure.Mcp.Tools.ApplicationInsights/src/Models/Query/ListTraceQueryResponse.cs

```diff
@@ -0,0 +1,17 @@
+namespace Azure.Mcp.Tools.ApplicationInsights.Models.Query;
+
+public class ListTraceQueryResponse
+{
+    public string? problemId { get; set; }
+    public string? target { get; set; }
+
+    public string? location { get; set; }
+
+    public string? name { get; set; }
+
+    public string? type { get; set; }
+
+    public string? operation_Name { get; set; }
```

Comment:
should be operationName

Reply:
This is a mapping object for conventional column mapping, and in the data, the operation name has been `operation_Name`.

I unterstand there's a level of inconsistency, introducing more mappings only gonna make debugging more complex.

Here's how the data looks like by KQL query:

<img width="730" height="433" alt="image" src="https://github.com/user-attachments/assets/ce73f361-6fe0-4908-adc9-58127b2dcd6e" />